src/Integration.UnitTests/Binding/CFamilyProjectBinderTests.cs

```diff
@@ -0,0 +1,125 @@
+﻿using System;
```

Comment:
Missing copyright...

Reply:
EVERY TIME. 😢 
We have to turn this on in sonarcloud...